sklearn/metrics/tests/test_regression.py

```diff
@@ -108,6 +117,7 @@ def test_regression_metrics_at_limits():
     assert_almost_equal(mean_squared_error([0.], [0.], squared=False), 0.00, 2)
     assert_almost_equal(mean_squared_log_error([0.], [0.]), 0.00, 2)
     assert_almost_equal(mean_absolute_error([0.], [0.]), 0.00, 2)
+    assert_almost_equal(pinball_loss([0.], [0.]), 0.00, 2)
```

Comment:
I would expect an exact match in this case
```python
    assert pinball_loss([0.], [0.]) == 0.00
```
or higher precision
```python
    assert_almost_equal(pinball_loss([0.], [0.]), 0.00, 10)
```

Reply:
+1 for the latter, just in case we get platform specific rounding errors.